crates/bevy_app/src/app.rs

```diff
@@ -88,9 +88,10 @@ impl Debug for App {
 }
 
 /// Each `SubApp` has its own [`Schedule`] and [`World`], enabling a separation of concerns.
```

Comment:
This needs better doc strings. Always has, but especially true if we're making this pub.